spark-task-sender/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang
+ 
+ADD src/sender /go/src/sender
+ADD config /go
+
+RUN go get github.com/Shopify/sarama
+RUN go install sender
+
+RUN echo "deb http://ppa.launchpad.net/webupd8team/java/ubuntu trusty main" > /etc/apt/sources.list.d/webupd8team-java.list
+RUN echo "deb-src http://ppa.launchpad.net/webupd8team/java/ubuntu trusty main" >> /etc/apt/sources.list.d/webupd8team-java.list
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EEA14886
+RUN apt-get -y update
+RUN /bin/echo debconf shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install oracle-java7-installer oracle-java7-set-default curl
+
+RUN curl -s http://d3kbcqa49mib13.cloudfront.net/spark-1.5.1-bin-hadoop2.6.tgz | tar -xz -C /usr/local/
```

Comment:
Now that you have the version as an environment variable, you should use it in place of "1.5.1" in the url. It is better if you move the ENV definition at the top of the file.